docs/commands/rhoas_service-registry_role_add.md

```diff
@@ -4,7 +4,13 @@ Add or update principal role
 
 ### Synopsis
 
-Add or update role for user or service account
+
+Add or update one of the following roles for a user or service account:
+
+* viewer (provides read access)
+* manager (provides read and write access)
+* admin (enables admin along with read and write access)
+
 
```

Comment:
I stumbled a bit reading the last line on admin access. You could use "provides admin access and read and write access), or trim it back to something like the following?

* viewer (read access)
* manager (read/write access)
* admin (admin and read/write access)

Reply:
@smccarthy-ie Thanks Steve -- updated now :)